content/en/blog/2025/announcing-etcd-3.6.md

```diff
@@ -0,0 +1,422 @@
+
+## Security
+
+etcd takes security seriously. To enhance software security in v3.6.0, we have improved our workflow checks by
+integrating `govulncheck` to scan the source code and `trivy` to scan container images. These improvements
+have also been backported to supported stable releases.
+
+etcd continues to follow the [Security Release Process][] to ensure vulnerabilities are properly managed and addressed.
+
+## Features
+
+### Migration to v3store
+
+The v2store has been deprecated since etcd v3.4 but could still be enabled via `--enable-v2`. It remained the source of
+truth for membership data. In etcd v3.6.0, v2store can no longer be enabled as the `--enable-v2` flag has been removed,
+and v3store has become the sole source of truth for membership data.
+
+While v2store still exists in v3.6.0, etcd will fail to start if it contains any data other than membership information.
+To assist with migration, etcd v3.5.18+ provides the `etcdutl check v2store` command, which verifies that v2store
+contains only membership data (see [PR 19113][]).
+
+Compared to v2store, v3store offers better performance and transactional support. It is also the actively maintained
+storage engine moving forward.
+
+The removal of v2store is still ongoing and is tracked in [issues/12913][].
+
+### Downgrade
+
+etcd v3.6.0 is the first version to fully support downgrade. The effort for this downgrade task spans
+both versions 3.5 and 3.6, and all related work is tracked in [issues/11716][].
+
+At a high level, the process involves migrating the data schema to the target version (e.g., v3.5),
+followed by a rolling downgrade.
+
+Ensure the cluster is healthy, and take a snapshot backup. Validate whether the downgrade is valid:
+
+```bash
+$ etcdctl downgrade validate 3.5
+Downgrade validate success, cluster version 3.6
+```
+
+If the downgrade is valid, enable downgrade mode:
+
+```bash
+$ etcdctl downgrade enable 3.5
+Downgrade enable success, cluster version 3.6
+```
+
+etcd will then migrate the data schema in the background. Once complete, proceed with the rolling downgrade.
+
+For details, refer to the [Downgrade-3.6] guide.
+
+### Feature Gates
+
+In etcd v3.6.0, we introduced Kubernetes-style feature gates for managing new features. Previously, we
+indicated unstable features through the `--experimental` prefix in feature flag names. The prefix was removed
+once the feature was stable, causing a breaking change. Now, features will start in Alpha, progress
+to Beta, then GA, or get deprecated. This ensures a much smoother upgrade and downgrade experience for users.
+
+See [feature-gates][] for details.
+
+### Livez/readyz checks
+
+etcd now supports `/livez` and `/readyz` endpoints, aligning with Kubernetes' Liveness and Readiness probes.
+`/livez` indicates whether the etcd instance is alive, while `/readyz` indicates when it is ready to serve
+requests. This feature has also been backported to release-3.5 since v3.5.11. See [livez/readyz][] for details.
```

Comment:
do we need to mention that we also backport it to v3.4.29? 

ref: https://github.com/etcd-io/etcd/pull/17128

Reply:
updated & thx